content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -180,19 +113,8 @@ problems:
     If the backend is unhealthy, check the pods in `istio-system`:
     * `kubectl get pods -n istio-system`
     * The `istio-ingressgateway-XX` pods should be running
-    * Check the logs of `backend-updater-0`, `ingress-bootstrap-XX`, `iap-enabler-XX` to see if there is any error
-
-1. Now that the certificate exists, the Ingress resource should report that it 
-  is serving on HTTPS:
-
-    ```
-    kubectl -n istio-system get ingress
-    NAME            HOSTS                                                        ADDRESS          PORTS     AGE
-    envoy-ingress   mykubeflow.endpoints.myproject.cloud.goog   35.244.132.159   80, 443   1d
-    ```
-
-    If you don't see port 443, look at the Ingress events using 
-    `kubectl describe` to see if there are any errors.
+    * Check the logs of pod `backend-updater-0`, `iap-enabler-XX` to see if there is any error
+    * Follow the steps [here](https://www.kubeflow.org/docs/gke/troubleshooting-gke/#502-server-error) to check the load balancer and backend service on GCP.
```

Comment:
Same comment as above I think this should be "You can follow the steps.."

Reply:
It's OK here too.